src/modules/flow/iio/nodes.c

```diff
@@ -26,13 +26,14 @@
 #include <sol-iio.h>
 
 
-#define ADD_CHANNEL(_scale, _offset, _channel, _nodename, _error) \
+#define ADD_CHANNEL(_scale, _offset, _channel, _nodename, _error) do { \
```

Comment:
Checkstyle complains here. Please, put the `do` on next line and indent accordingly.